extensions/blocks/image-compare/view.scss

```diff
@@ -0,0 +1,337 @@
+/**
+ * Juxtapose - v1.2.0 - 2017-12-18
```

Comment:
Latest is from 2019 1.2.1:
 — if to fork now here, why not take the latest? 

https://github.com/NUKnightLab/juxtapose/blob/94da426b3906f1d305a89d115f8fa3da67827d0c/build/js/juxtapose.js#L1

Reply:
The last commit is adding Google Tag Manager support: https://github.com/NUKnightLab/juxtapose/commits/94da426b3906f1d305a89d115f8fa3da67827d0c/build/js/juxtapose.js

So we're not missing anything, and not what we want. There has also been a fair amount of modification to the library, to modernize and remove features such as a bunch of weird built-in Flickr stuff.